application/frontend/config/main.php

```diff
@@ -30,14 +33,18 @@
             // http://www.yiiframework.com/doc-2.0/guide-rest-routing.html
             'rules' => [
                 [
+                    // http://www.yiiframework.com/doc-2.0/yii-rest-urlrule.html
                     'class' => 'yii\rest\UrlRule',
-                    'controller' => ['user', 'authentication'],
+                    'controller' => ['authentication', 'user'],
+                    'extraPatterns' => [
```

Comment:
while both controllers are in the mix and getting a bunch of standard patterns automatically generated for each of them from `UrlRule`, the `AuthenticationController` doesn't implement a `view` so even if an "authorized" requester were to issue a `GET /authentication/abc123`, the response will still be a `400` because of `BaseRestController`, specifically in this case, it will fail because there was no `id` passed along.